collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -355,7 +355,7 @@ public void onClick(View v) {
             if (intent != null) {
                 Uri uri = intent.getData();
 
-                if (getContentResolver().getType(uri).equals(InstanceColumns.CONTENT_ITEM_TYPE)) {
+                if (null != uri && getContentResolver().getType(uri).equals(InstanceColumns.CONTENT_ITEM_TYPE)) {
```

Comment:
You'll see that we typically put the thing that changes (`uri` in this case) before the thing that doesn't (`null` in this case) when doing comparisons. It's a convention I didn't realize we had but you'll see it's very consistent across the code base!